schemas/.markdownlint.yaml

```diff
@@ -0,0 +1,14 @@
+# Unordered list indentation
+MD007:
+    indent: 2
+
+MD012: false # allow blank lines
+
+MD013:
+    line_length: 800
+    tables: false
+
+MD024: false # duplicate headings
+MD032: false # blanks-around-lists Lists should be surrounded by blank lines
+MD033: false # inline HTML
+MD047: false # single-trailing-newline Files should end with a single newline character
```

Comment:
Rules 32 and 47 are disabled because they didn't pass. We can enable them again and format the file if that is what we desire.

Reply:
Yes, please use the same rules for all `*.md` files and change the files that don't follow the rules yet.

Reply:
I'm not sure I agree that the same rules apply for specification documents and our internal project documentation in markdown files. I think we already rejected a pull request that applied the same standards to both - We need to be strict and careful with the documents that are published, I'm not sure the same applies if people are adding different content types.

Reply:
HI @ralfhandl , I am not against in what you are proposing but @lornajane is right about the rejected PR. Please check https://github.com/OAI/Overlay-Specification/pull/105

Reply:
Given that the commented-out rules are trivial to observe, I'd use the same rules. 

The more interesting case will be when we make the rules for new spec documents stricter and will need different rule sets for files in the `versions` folder, as we already do for OAS.

Reply:
@ralfhandl Ok. But what this means for the other comments that you made? Such as https://github.com/OAI/Overlay-Specification/pull/115#discussion_r1900669324

To be more precise, are we keeping the multiple lint configs or are we using a single one?

Reply:
I prefer keeping a single rule file and adjusting all Markdown files across all folders accordingly.